packages/components/src/internal/components/pipeline/PipelineJobsPage.tsx

```diff
@@ -53,6 +54,7 @@ export class PipelineJobsPageImpl extends React.PureComponent<Props & InjectedQu
             baseFilters,
             sorts: [new QuerySort({ fieldKey: 'Created', dir: '-' })],
             requiredColumns: ['Provider'],
+            containerFilter: getContainerFilter()
```

Comment:
Hmm, why isn't this already happening? These models should be defaulting to the desired containerFilter if projects is enabled.

Reply:
You are right this is not needed..